tests/integration/test_branch_integration.py

```
@@ -0,0 +1,369 @@
+            if path.exists():
+                with open(path, 'r') as f:
+                    workflows[wf_file] = yaml.safe_load(f)
+        
+        return workflows
+    
+    def test_all_workflows_use_consistent_action_versions(self, all_workflows):
+        """Verify same actions use consistent versions across workflows."""
+        action_versions = {}
+        
+        for wf_file, workflow in all_workflows.items():
+            for job_name, job in workflow.get('jobs', {}).items():
```

Ruff (0.14.7)

Loop control variable `job_name` not used within loop body

Rename unused `job_name` to `_job_name`

(B007)